OneSignalSDK/unittest/src/test/java/com/onesignal/InAppMessagingHelpers.java

```diff
@@ -95,6 +95,17 @@ private static JSONObject basicIAMJSONObject(final JSONArray triggerJson) throws
         return json;
     }
 
+    public static OSTestInAppMessage buildTestMessageWithLiquid(final JSONArray triggerJson) throws JSONException {
+        JSONObject json = basicIAMJSONObject(triggerJson);
+        json.put("has_liquid", true);
```

Comment:
Can you add has_liquid to a constant? I know other constants aren't put on constants, but we can start doing so 😬

Reply:
done!